### DISCUSSION TITLE
Where is Slack?

### DISCUSSION THREAD

**Discussion by awmiller** [The-OpenROAD-Project/OpenLane#1688] [General]
Slack link not working... thanks

**Comment by vijayank88** [comment]
https://invite.skywater.tools/ link is working. Once you registered you can able to access the below 
https://open-source-silicon.slack.com/

**Reply by eusanchez** [reply]
Is not working... Can you update the link?

**Reply by vijayank88** [reply]
Check this link: https://theopenroadproject.org/ and click on #Slack

**Reply by bodlacyn** [reply]
It is not working... Can you update the link?
